dev/release/verify-release-candidate.bat

```diff
@@ -140,7 +140,7 @@ set PYARROW_WITH_PARQUET=1
 set PYARROW_WITH_PARQUET_ENCRYPTION=1
 set PYARROW_WITH_DATASET=1
 set PYARROW_TEST_CYTHON=OFF
-python setup.py build_ext --inplace --bundle-arrow-cpp bdist_wheel || exit /B 1
+python -m build -C--bundle-arrow-cpp . --wheel || exit /B 1
```

Comment:
Can we use a long option (`--config-setting`?) instead of `-C` for readability?

Can we use `python -m build . ${OPTIONS}` or `python -m build ${OPTIONS} .` instead of `python -m build ${OPTIONS1} . ${OPTIONS2}` for readability?

Reply:
Done!